Example/Storage/Tests/Integration/FIRStorageIntegrationTests.m

```diff
@@ -454,6 +454,86 @@ - (void)testCancelDownload {
     [self waitForExpectations];
 }
 
+- (void)testUpdateMetadata {
+  XCTestExpectation *expectation =
+    [self expectationWithDescription:@"testUpdateMetadata"];
+
+  FIRStorageReference *ref = [self.storage referenceWithPath:@"ios/public/1mb"];
+
+  // Update all available metadata
+  FIRStorageMetadata *metadata = [[FIRStorageMetadata alloc] init];
+  metadata.cacheControl = @"cache-control";
+  metadata.contentDisposition = @"content-disposition";
+  metadata.contentEncoding = @"gzip";
+  metadata.contentLanguage = @"de";
+  metadata.contentType = @"content-type-a";
+  metadata.customMetadata = @{ @"a" : @"b" };
+
+  void (^assertMetadata)(FIRStorageMetadata*, NSString*, NSDictionary*) =
```

Comment:
I wanted this to be part of the test method. I moved this and the last assert of this test out to be their own function.